### PR TITLE
chore: fix comment

### PR DESCRIPTION
**Pull request by cvarley100**
## What, How & Why?

Fixes an incorrect comment.

## ☑️ ToDos
* [ ] 📝 Changelog entry
* [ ] 📝 `Compatibility` label is updated or copied from previous entry
* [ ] 🚦 Tests
* [ ] 📝 Public documentation PR created or is not necessary
* [ ] 💥 `Breaking` label has been applied or is not necessary

* [ ] typescript definitions file is updated
* [ ] jsdoc files updated
* [ ] Chrome debug API is updated if API is available on React Native

### CODE REVIEW

File: lib/index.d.ts [hunk 1]
```diff
@@ -192,8 +192,8 @@ declare namespace Realm {
     }
 
     /**
-     * RealmJsonSerializeReplacer solves circular structures when serializing Realm entities
-     * @example JSON.stringify(realm.objects("Person"), Realm.RealmJsonSerializeReplacer)
+     * JsonSerializeReplacer solves circular structures when serializing Realm entities
+     * @example JSON.stringify(realm.objects("Person"), Realm.JsonSerializeReplacer)
```
Comment (steffenagger) [inline lib/index.d.ts:196]:
```suggestion
     * JsonSerializationReplacer solves circular structures when serializing Realm entities
     * @example JSON.stringify(realm.objects("Person"), Realm.JsonSerializationReplacer)
```

**Review by steffenagger — CHANGES_REQUESTED** [review]
First of all, thanks for letting us know, and the PR!

Unfortunately I can see I made several errors in this comment (my apologies).
The current change is not quite enough, as it actually should be `JsonSerializationReplacer`.

I've added a change-suggestion (just to visualize).
If you don't want any more hassle, let me know.

**Comment by cvarley100** [comment]
@steffenagger My bad, looks good!

**Review by steffenagger — APPROVED** [review]
Thanks for reporting and supplying a fix 🙂

**Pull request merged**